client/components/Navbar.tsx

```
@@ -54,9 +58,56 @@ const Navbar = () => {
+
+  if (isLoading) {
+    userContent = (
+      <span className="text-lg text-gray-700">Loading User...</span>
+    );
+  } else if (user) {
+    userContent = (
+      <div className="relative">
+        <button
+          onClick={() => setDropdownOpen(!dropdownOpen)}
+          className="flex items-center gap-2 text-lg text-gray-700 hover:text-green-700"
+        >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)